site/docs/patterns/header-block.mdx

```diff
@@ -0,0 +1,148 @@
+---
+title: Header block
+layout: DetailPattern
+tags:
+  - pattern
+aliases:
+  - /salt/patterns/header-block
+data:
+  resources:
+    [
+      { href: "https://storybook.saltdesignsystem.com/?path=/story/patterns-header-block--header-block", label: "Header block examples" },
+    ]
+  components: ["Dialog", "Drawer", "Overlay"]
+---
+
+Header block is a standardized header for container components and application regions, used to display various text elements depending on scenarios. Headings can also display accent and status to highlight messaging.
+
+<Diagram
+  src="/img/patterns/header-block/hero-shot.png"
+  alt="Example header block. Displaying close action, status, preheader and description."
+  border
+/>
+
+## When to use
+
+Use this pattern to:
+
+- Display a combination of headings, pre-headers and descriptions within container components such as [Dialog](/salt/components/dialog), [Drawer](/salt/components/drawer) and [Overlay](/salt/components/overlay). This can help provide clear context and purpose of a component or interaction within a UI.  
+**Note:** some Salt components, such as Dialog and Overlay, have header block embedded in by default, across both code and figma.
+- Maintain a visually consistent layout and presentation for headers, and establish visual hierarchy of headings with supplementary text across different components and content layouts.
+- Display headings with status icons to communicate different statuses, depending on the context of the component the header block sits within.
+
+## How to build
+
+### Anatomy
+
+The header block pattern comprises the following visual and text elements:
+
+1. **Accent bar**: spans the height of, and brings attention, to the heading and supplementary text.
+2. **Heading**: serves as the primary title of a content type.  
+3. **Pre-header**: displayed above the main heading. Used to introduce a section or draw attention to important information, such as a grouping or step in a process.  
+4. **Description**: displayed below the main heading as a secondary text element that provides additional information or context to the main title. Brief descriptions can be used to clarify the main title, or provide further details about the content of the design.
+5. **Close button**: optional action for the user to dismiss a component or element.  
+6. **Status icon**: visual indicator that communicates status.
+
+<Diagram
+  src="/img/patterns/header-block/header-block-anatomy.png"
+  alt="A header block anatomy."
+  border
+/>
+
+### Layout
+
+Header block can be configured to be displayed using three different sizes, each differentiated by the size of the padding and the heading type. We recommend using an appropriately sized header block, relative to the container element it is embedded within.
```

Comment:
Header block is a pattern so can't be configured

Reply:
@bhoppers2008 "Header block can ~~be configured to~~ be displayed" ?